pyproject.toml

```diff
@@ -261,7 +258,7 @@ markers = ["requires_netcdf: marks tests that require netcdf files to run"]
 [tool.ruff]
 src = ["src/xscen"]
 line-length = 150
-target-version = "py39"
+target-version = "py310"
```

Comment:
You can simply remove this line. `ruff` will know to look at the lowest supported version.
```suggestion
```